google-api-credentials.json

```diff
@@ -0,0 +1,12 @@
+{
+  "type": "service_account",
+  "project_id": "census-2020-1574478662266",
+  "private_key_id": "5c446a2e5e228710a947562efbbb68ea96360c0e",
+  "private_key": "-----BEGIN PRIVATE KEY-----\nMIIEvQIBADANBgkqhkiG9w0BAQEFAASCBKcwggSjAgEAAoIBAQCmx0tMjcryOmA/\n3K73B7+ABsV2tMSBq36sQSJhsdxgUQhgKd+c5kKnTn6QCD1xHFdWgnV0JQ/XEaVs\nRPrRqUHK5tQY3NaGMXFQWKXjAaVXPGCDhVi5fkhgjwelGbqN7nduFcj5ETNB7am5\n6o1bMywT4ofYucgfkEdetyrtZi8ycI9L/NzShgAz3xmK1kn+iaIuZaOOCqXsXSDV\nmmrKw5S6eE1XApOjZy07eWyLmbcsfiN26zDysp0ExleOn9ouOJ8wkemKvzrlmy+I\nFT1JrjzRjT1+CDRc5RkJpzYSAxkozmDeFA9k8VBZuMf7OvxIMkveQ76jOFcA6V/Q\nQG1Dv/NrAgMBAAECggEAFeXjCCkx9X4etyPlwT++MDQzZzOy/Yej+UAQIJ223tqk\n6+NznelJ5SfUTFsYnipneU9l1psrKkUB4GZqPG8p2VdXj/AnJrWFOFR7a4QCzqJ5\n1asDjE99d1gwW8g2C369JbIski9k+IoZF/UpHElqxy2eQqGaGjLafCA2vbcGCWwx\naRJChjXhYmKZmvuilduHq0DlGDQBf8GHNVeOSWUsWDHM/s+ZS6avbl7NG277Esks\nqfgkmBfR0iUDlC9h8DgI9wqBdrv1tSrLDyNODON++Ymgpo8qN6I1mzA3vEQf06VG\nqwJ91IpsZ9gNEJeHIH0qcNIaBfczjaXi7X+4gDr98QKBgQDpkz0y67Atx3yPIy9G\nW/sKxzbbqM4apRnWJFL5jMNLK2pSCi1OFi61ehbpZ4356SVZE6fSwRkhtHAa5gsu\nBzvWJ3AmzpDXXmx8q1LKw7wQpDJJq73ws7hTgK0jtx6MSJ71bBsyo5a0Ow2dszAp\noVHT0RZ0mNtM1KmwqvE6VoG94wKBgQC2yldAsPIYvqyeaM41PlrmVi26rvggqTMU\nOU3Cq3fNKZowPLh4ar2ggu6uRYMcj0qlWDOsQEinMkOuW48Lx/ddhDG4s7/S9+QM\ngGd+3SCd2fo6pqbxUYStGS5YUCCc8pj2wbKggqISP+pqq9264YkQq4zwyb1Lt4af\nAnt/NSJq2QKBgEMB02+1P5CShwBFNr23i3hXGDzB24a3UIcnWopuwGPjK5gnhUG+\niV3UjZgd9hP4t5crVVzXuo75xaNh1Vk6CAdyvvDTF6hBVVwahiDXMC+R+z85Pr87\nv6hs5L63PNM/ziJYBiFr3sMLoedYzXtT5Taf6BJA+8C5zYh6McPrp0LDAoGAUq2E\n97rRVkXweBo5ozWYEfVK69S2YLeHZg9hT7sVDhRSLJ1aZwUIznBKm0LPwfxCLDWP\n8cqF0jJXEKdTNWX2SjM/9iY/jqIXYVw289f7J8rvdtXoTuCrXUw1fTaI+dR1Qa9t\nJ1LPEegdFLC2MxBOWd7uef8WAwh4+LU7vNcAwUkCgYEA3HAErOc8GPnb4nXGoJGA\nkmNTDcfyQH5hLt9dnk7RPSlPX7Qx9Noru1HY5OwY6USZki/nAA+kcLZQ85KCNsaT\n7gu9B593RrRZe/qmHZeubRCRFj59bBfhbmKcYT516PFHkYhw90T6SgmBXHDX4Ec9\nqtw4V2q0qqAHA/tzuQJ7IIM=\n-----END PRIVATE KEY-----\n",
```

Comment:
is this suppose to be here?

Reply:
Good catch; no! But I removed the key from Google, so it no longer works.